trackthenews/core.py

```diff
@@ -271,6 +339,42 @@ def config_twitter(config):
 
     return config
 
+
+def config_mastodon(config):
+    mastodon_setup = input("Would you like the bot to post to Mastodon? (Y/n) ")
+    if mastodon_setup.lower().startswith('n'):
+        return config
+
+    if 'mastodon' in config.keys():
+        replace = input("Mastodon configuration already exists. Replace? (Y/n) ")
+        if replace.lower() in ['n','no']:
+            return config
+
+    api_base_url = input("Enter your Mastodon instance URL (e.g., 'https://mastodon.social'): ")
+    access_token = input("Enter your access token: ")
```

Comment:
It would be nice to add some more instruction here, maybe a URL directly to the configuration page?

Additionally I ran into an issue the first time I ran this because c/p from the Mastodon interface left spaces on either side of the token. It would be nice to strip spaces before attempting to authenticate and saving.

Reply:
Good point. I've added `.strip()` to all user inputs, and added more instructions.